management-account/terraform/sso-scim.tf

```diff
@@ -8,3 +8,11 @@ module "scim" {
   sso_identity_store_id = local.sso_admin_identity_store_id
   not_dry_run           = true
 }
+
+module "entraid_scim" {
+  # tflint-ignore: terraform_module_pinned_source
+  source              = "github.com/ministryofjustice/moj-terraform-scim-entra-id"
+  azure_tenant_id     = "your-tenant-id"
+  azure_client_id     = "your-client-id"
+  azure_client_secret = "your-client-secret"
```

Comment:
Presuming these are placeholders for testing that everything goes through okay before worrying about injecting the actual secrets, but just thought I'd double-check 🙈

Reply:
Yes, they are placeholders, correct. Final version will read these values from secrets manager as the github-scim lambda does.